package.json

```diff
@@ -0,0 +1,49 @@
+{
+  "name": "todoapp",
+  "version": "1.0.0",
+  "description": "A simple To-do list app build with Vanilla JS and Webpack",
+  "main": "index.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "start": "webpack-dev-server --env production --open",
```

Comment:
- [ ] Also kindly change the start command script to simply this:
`    "start": "webpack-dev-server --open",
`